### PR TITLE
first push for review

### PR DESCRIPTION
**Pull request by JosephLarbi21**
# What?

# Why?

# How?

### CODE REVIEW

**Review by Haywayaheadshot — CHANGES_REQUESTED** [review]
Hi @joey2022-eng , 

You have done very well coming this far with this project! There are a lot of issues to fix but we will tackle them one at a time. 

- First of all, please re-organize your files. Take for example [this](https://www.youtube.com/watch?v=s_Ie_yh_4Co&list=PL0Zuz27SZ-6PA3nkkEAr91n4bK-NunwPU) video. If you skip to 2:14:40, you will realize on the left, all the files have been organized in their appropriate folders.

Please tackle this issue first

**Comment by Haywayaheadshot** [comment]
@joey2022-eng Also, please note that if you do not understand anything, you can always ask. I will be available to reply under this thread.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <li><a href="Landing Page/Service.html">Service</a></li>
+      <li><a href="#">Lives</a></li>
+      <li><a href="Landing Page/about.html">About</a></li>
+      <li><a href="Landing Page/contact.html">Contact</a></li>
+    </ul>
+
+    <script>
+      let menubox = document.getElementById("menubox")
+      let menuicon = document.getElementById("menuicon")
+
```
Comment (Haywayaheadshot) [inline index.html:70]: Also, it is not a good practice to leave commented codes in a project. You can check out [this](https://www.w3schools.com/js/js_best_practices.asp) article to understand it more. 

Please delete all unused codes.

**Comment by JosephLarbi21** [comment]
> Hi @joey2022-eng ,
> 
> You have done very well coming this far with this project! There are a lot of issues to fix but we will tackle them one at a time.
> 
> * First of all, please re-organize your files. Take for example [this](https://www.youtube.com/watch?v=s_Ie_yh_4Co&list=PL0Zuz27SZ-6PA3nkkEAr91n4bK-NunwPU) video. If you skip to 2:14:40, you will realize on the left, all the files have been organized in their appropriate folders.
> 
> Please tackle this issue first

Please am done with the changes you ask me to do.
The only issue right now is how to commit and push

> Hi @joey2022-eng ,
> 
> You have done very well coming this far with this project! There are a lot of issues to fix but we will tackle them one at a time.
> 
> * First of all, please re-organize your files. Take for example [this](https://www.youtube.com/watch?v=s_Ie_yh_4Co&list=PL0Zuz27SZ-6PA3nkkEAr91n4bK-NunwPU) video. If you skip to 2:14:40, you will realize on the left, all the files have been organized in their appropriate folders.
> 
> Please tackle this issue first

Please am done with it.

**Comment by JosephLarbi21** [comment]
> Hi @joey2022-eng ,
> 
> You have done very well coming this far with this project! There are a lot of issues to fix but we will tackle them one at a time.
> 
> * First of all, please re-organize your files. Take for example [this](https://www.youtube.com/watch?v=s_Ie_yh_4Co&list=PL0Zuz27SZ-6PA3nkkEAr91n4bK-NunwPU) video. If you skip to 2:14:40, you will realize on the left, all the files have been organized in their appropriate folders.
> 
> Please tackle this issue first

Please am done with the re-organization.

**Comment by JosephLarbi21** [comment]
Done re-organizing it

**Pull request closed without merging**